workos/portal.py

```diff
@@ -16,6 +17,47 @@ def request_helper(self):
             self._request_helper = RequestHelper()
         return self._request_helper
 
+    def create_organization(self, organization):
```

Comment:
Curious if Organizations are going to be used for anything else in the future. If so, would it make sense to put it into a Portal module or should it go somewhere else?

Reply:
We had this discussion as we kicked things off and it was decided that while it's not the best future home it is the one most consistent with the approach we've taken so far. In the future we might re-organize things to a more resource based approach, but that would have enough breaking changes we'd want it to be a major version.